tensorflow/python/util/serialization.py

```diff
@@ -42,6 +42,10 @@ def get_json_type(obj):
   # e.g. optimizer, layer
   if hasattr(obj, 'get_config'):
     return {'class_name': obj.__class__.__name__, 'config': obj.get_config()}
+  
+  # Serialize EagerTensors (fix for saving 1.x Keras Models)
+  if isinstance(obj, tf.python.framework.ops.EagerTensor):
```

Comment:
You need to create a new test for this feature at https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/tests/serialization_util_test.py